sim/battle.ts

```diff
@@ -148,6 +149,7 @@ export class Battle {
 		this.zMoveTable = {};
 		this.maxMoveTable = {};
 		this.trunc = this.dex.trunc;
+		this.clampIntRange = this.dex.clampIntRange;
```

Comment:
```suggestion
		this.clampIntRange = Utils.clampIntRange;
```

You'll need to import `Utils`, of course.

Reply:
Isn't sim supposed to be self-contained?

Reply:
You should be deleting `clampIntRange` in Dex, too.

Reply:
I can do this but i was under the impression outside imports were bad.